src/net_processing.cpp

```diff
@@ -3717,6 +3743,8 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
             return;
         }
 
+        SetupAddressRelay(pfrom, *peer);
```

Comment:
`if (!pfrom.IsInboundConn() || !SetupAddressRelay(pfrom, *peer)) { error; return; }` might have been better so you don't have to keep the knowledge that SetupAddressRelay always returns true for inbound conns in your head.

Reply:
makes sense, I'll change if I have to push again

Reply:
`Assume(Setup())` would do that?

Reply:
addresses in #22618